training-react/src/app/components/App/screens/App/screens/UserEdit/index.js

```diff
@@ -21,27 +18,7 @@ class UserEdit extends Component {
   };
 
   render() {
-    const { handleSubmit } = this.props;
-
-    return (
-      <div className={styles.desc}>
-        <h2 className={styles.title}>Edit</h2>
-        <form onSubmit={handleSubmit(this.onSubmit)}>
-          <Field name="username" component={customInput} type="text" label="Username" />
-          <Field name="fullname" component={customInput} type="text" label="Fullname" />
-          <Field name="gender" component={customInput} type="text" label="Gender" />
-          <Field name="country" component={customInput} type="text" label="Country" />
-          <Field
-            name="mail"
-            component={customInput}
-            type="text"
-            label="Mail"
-            validate={[required, isEmail]}
-          />
-          <button type="submit">Submit</button>
-        </form>
-      </div>
-    );
+    return <UserEditDetails onSubmit={this.onSubmit} {...this.props} />;
```

Comment:
`{...this.props}` could be inefficient as you are passing every prop to the bumb component, making it update even if any of those props aren't being used.
It's better to pass just the props you need or even every prop one by one.

Reply:
Important comment :+1: